shell/middleware/authenticated.js

```diff
@@ -146,41 +143,8 @@ function invalidResource(store, to, redirect) {
 }
 
 export default async function({
-  route, app, store, redirect, $cookies, req, isDev, from, $plugin, next
+  route, store, redirect, from, $plugin, next
 }) {
-  if ( route.path && typeof route.path === 'string') {
-    // Ignore webpack hot module reload requests
-    if ( route.path.startsWith('/__webpack_hmr/') ) {
-      return;
-    }
-
-    // Ignore the error page
-    if ( route.path.startsWith('/fail-whale') ) {
-      return;
-    }
-  }
-
-  // This tells Ember not to redirect back to us once you've already been to dashboard once.
-  if ( !$cookies.get(REDIRECTED) ) {
-    $cookies.set(REDIRECTED, 'true', {
-      path:     '/',
-      sameSite: true,
-      secure:   true,
-    });
-  }
-
-  const upgraded = route.query[UPGRADED] === _FLAGGED;
-
-  if ( upgraded ) {
-    app.router.applyQuery({ [UPGRADED]: _UNFLAG });
-
-    store.dispatch('growl/success', {
-      title:   store.getters['i18n/t']('serverUpgrade.title'),
-      message: store.getters['i18n/t']('serverUpgrade.message'),
-      timeout: 0,
-    });
-  }
```

Comment:
Moved to index and we can see it still works.


https://github.com/rancher/dashboard/assets/55104481/c1abd829-1715-4c5a-b7a0-e3f9dab2479c